app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/triage/CommonTriageQuestionsController.scala

```diff
@@ -439,6 +470,13 @@ class CommonTriageQuestionsController @Inject() (
   private def howManyPropertiesBackLink(
     state: Either[StartingNewDraftReturn, FillingOutReturn]
   ): Option[Call] = {
+
+    val isFurtherReturn = state
```

Comment:
`isFurtherReturn` in `FillingOutReturn` already takes take of the case when it's a representative - you don't need this `fold` and you probably don't need this `val` then